projects/js-packages/publicize-components/src/components/manage-connections-modal/confirmation-form/index.tsx

```diff
@@ -199,6 +199,14 @@ export function ConfirmationForm( { keyringResult, onComplete, isAdmin }: Confir
 							'jetpack'
 						) }
 					</p>
+					{ keyringResult && keyringResult.show_linkedin_warning && (
+						<p className={ styles[ 'header-text' ] }>
+							{ __(
+								`If you want to connect LinkedIn company pages and do not see them, please try again after 5 minutes.`,
+								'jetpack'
+							) }
```

Comment:
Should we use a notice component here so the warning is more visible?
![image](https://github.com/user-attachments/assets/289a84b0-f7c0-4c78-a8ab-06515ad21439)

Reply:
Yeah, that may be better.